content/faq.md

```diff
@@ -0,0 +1,399 @@
+
+## Arvo
+
+### What is Arvo? {#arvo-1}
+
+Arvo is the Urbit operating system. Unlike conventional operating systems, it’s functional and deterministic. Arvo is written in Hoon, our purely functional programming language.
+
+Instances of the Arvo operating system, called ships, communicate as peers on what we refer to as “the Arvo network.” The Arvo network succeeds where other peer-to-peer networks have failed. Using Azimuth, Arvo ships can prove their identities to one another.
+
+### What is unique about Arvo? {#arvo-2}
+
+Arvo is much different from other operating systems because of its determinism. Processing in the system happens in a unique way: when an event happens, a transition function takes that event and the old state of Arvo, and then produces an effect and a new state of Arvo. To visualize:
+
+[event, old state] -> [effects, new state]
+
+Thus, all events are permanently logged in the system, and you can always go back and see the state of the system at any point in time.
+
+### How is Arvo connected to Ethereum? {#arvo-3}
+
+When an Arvo ship (instance) is started for the first time, you must use a key-file containing the private keys for your Azimuth point's networking keys. Azimuth uses the Ethereum blockchain as its decentralized ledger.
+
+See our [Getting Started](@/docs/getting-started/_index.md) guide to learn how to get your ship onto the Arvo network.
+
+### Will the Arvo network survive if Ethereum dies? {#arvo-4}
+
+Yes. It would be annoying, but Azimuth would be ported to another kind of decentralized ledger.
+
+### How do I install Arvo? {#arvo-5}
+
+Check out our guides [here](@/docs/getting-started/_index.md)
+
+### How do I use Arvo? {#arvo-6}
+
+The [Using](@/docs/using/_index.md) section of the documentation will help you out.
+
+### How do over-the-air updates work? {#arvo-7}
+
+Your sponsor, a star or galaxy that your ship “lives” under, may send you new source code for your system. When you receive new source code from your sponsor, your system recompiles itself using that code, performs any necessary data migrations, and keeps running. Ideally this happens seamlessly without the user even noticing, although there is sometimes a slowdown while rebuilding the system from source.
+
+### What is Landscape? {#arvo-8}
+
+Landscape is the Arvo web interface that includes social functions such as chat and message boards. You can access Landscape by navigating to <your-planet>.arvo.network, where <your-planet> is your running ship that’s connected to the Arvo network.
+
```

Comment:
So, an open question: Isn't Landscape about to be retired in favour of Chat and Publish? Like, very, very soon? Or are we still calling those things Landscape in Modulo?

Reply:
That's correct. We should probably remove stuff about Landscape from this.